### PR TITLE
improve console docs: collapse org pages, and project/stack pages from 4 to 2

### LINKED ISSUE #346: reorganize org, project, and stack console docs
## overview
in follow up to... https://github.com/pulumi/pulumi-hugo/pull/307

the breakdown of pages in the console docs is not consistent and is a setup for (1) repeated information and (2) it not being clear what page users should go to in order to answer their questions

I had also called this out in the pr description of #21

## design details
- [x] move organization roles content to the organizations page
    - [x] delete org roles page, setup redirect
    - [x] remove duplicated content, reduce content as possible, and present information in a logical order
- [x] move stack permissions content to the projects and stacks page
    - [x] update heading to be projects and stacks, instead of project and stack management; setup redirect
    - [x] remove duplicated content, reduce content as possible, and present information in a logical order

### PR DESCRIPTION
**Pull request by susanev**
## overview
this is follow-on work from #21 and fixes #346
this work was originally completed in https://github.com/pulumi/pulumi-hugo/pull/307 but @cnunciato asked me to split that work up into multiple prs, the first two have merged: https://github.com/pulumi/pulumi-hugo/pull/348 and https://github.com/pulumi/pulumi-hugo/pull/366. this is the last left-over work from 307.

as far as the larger body of work, the last thing i will do sometime soon is a pass at updating all of our screenshots: #347

this is collapsing the two org pages into one and the two project/stack pages into one. this will make it easier for folks to find the page with the answer to their question as well as reduce the need for us to duplicate content across pages.

➡️   **super important note** ⬅️ 
im not adding any new content, only moving and cleaning up content.
files changed can be difficult to review when moving content. if possible i suggest scanning the current pages: [organizations](https://www.pulumi.com/docs/intro/console/organizations/), [organization-roles](https://www.pulumi.com/docs/intro/console/organization-roles/), [project-and-stack-management](https://www.pulumi.com/docs/intro/console/project-and-stack-management/), and [stack-permissions](https://www.pulumi.com/docs/intro/console/stack-permissions/) and then scanning the two collapsed pages in the preview: organizations, and projects-and-stacks to get some bearings before starting review.

### CODE REVIEW

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (susanev) [inline themes/default/content/docs/intro/console/projects-and-stacks.md:149]:
```suggestion
## Deleting a Stack
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [inline themes/default/content/docs/intro/console/organizations.md:18]:
```suggestion
## Organization Pages
```

File: themes/default/content/docs/intro/console/teams.md
Comment (susanev) [inline themes/default/content/docs/intro/console/teams.md:49]:
```suggestion
## Creating a Team{#creating-a-team}
```

File: themes/default/content/docs/intro/console/teams.md
Comment (susanev) [inline themes/default/content/docs/intro/console/teams.md:64]:
```suggestion
## GitHub-based Teams
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [inline themes/default/content/docs/intro/console/organizations.md:105]:
```suggestion
[GitLab Groups](https://docs.gitlab.com/ce/user/group/)
```

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (cnunciato) [inline themes/default/content/docs/intro/console/projects-and-stacks.md:56]:
```suggestion
![Organization projects and stacks](/images/docs/reference/service/organization-stacks.png)
```

File: themes/default/content/docs/intro/console/stack-permissions.md
Comment (cnunciato) [inline themes/default/content/docs/intro/console/stack-permissions.md:13]: This is a good introductory line that it'd be nice to keep if we could. Right under the `Stack Permissions` heading would probably be a good place for it.

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (cnunciato) [inline themes/default/content/docs/intro/console/projects-and-stacks.md:53]: We seem to have lost the Assigning Stack Permissions section: https://www.pulumi.com/docs/intro/console/stack-permissions/#assigning-stack-permissions

Any reason not to bring that forward?

**Review by cnunciato — COMMENTED** [review]
Love the direction of this. Still some work to do to finish it off, but definitely like where it's headed.

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [reply themes/default/content/docs/intro/console/organizations.md:41]: this is actually something that still needs to be documented, there's an issue for it: #15 i can add that into the follow on work.

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (susanev) [reply themes/default/content/docs/intro/console/projects-and-stacks.md:53]: nope, that was a mistake

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [reply themes/default/content/docs/intro/console/organizations.md:46]: i will do a follow on pr for adding this content, im trying to limit this pr to not include new content.

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [reply themes/default/content/docs/intro/console/organizations.md:52]: i can change this to menu, but am trying to avoid directional words. we really need a label on it in the ui.

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [reply themes/default/content/docs/intro/console/organizations.md:48]: i changed this because i found switching confusing, though also not convinced moving makes it any more clear. so i can revert this change.

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (susanev) [reply themes/default/content/docs/intro/console/projects-and-stacks.md:53]: oh sorry i misread this, i thought you were commenting on the table.

Comment (susanev) [reply themes/default/content/docs/intro/console/projects-and-stacks.md:53]: this content is now in the stack permissions section, and on the team page

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [reply themes/default/content/docs/intro/console/organizations.md:58]: I dont believe this needs a paragraph explaining what roles are, but if you would like to draft one i can add it.

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [reply themes/default/content/docs/intro/console/organizations.md]: i dont understand what you are communicating here, can you provide a specific suggestion for what to change?

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (susanev) [reply themes/default/content/docs/intro/console/projects-and-stacks.md]: can you write a summary paragraph that you would like included here?

File: themes/default/content/docs/intro/console/organizations.md
Comment (chrsmith) [inline themes/default/content/docs/intro/console/organizations.md:61]: Nit: Should "Inviting members" be capitalized? I don't believe you are supposed to capitalize the first word after a colon, but I reserve the right to be embarrassingly wrong here.

**Review by chrsmith — APPROVED** [review]
I didn't look too closely at the specific content, but if it is just rearranging the docs we already have, I think this LGTM. Having fewer, larger pages is probably a net improvement.

However, IMHO, it would be helpful if the outline on the right side were a little easier to follow. e.g. maybe a larger indentation between headings and its sub headings? But we shouldn't block on that.

image

**Comment by susanev** [comment]
@chrsmith yah i will not be changing the presentation of the right panel in our docs in this work, if you'd like that work tracked please open a new issue!

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (jkisk) [inline themes/default/content/docs/intro/console/projects-and-stacks.md:17]: I like this section, this was unclear to me when I started with Pulumi.

File: themes/default/content/docs/intro/console/organizations.md
Comment (jkisk) [inline themes/default/content/docs/intro/console/organizations.md:98]: Perhaps we should say "Pulumi will not have access..." just to be extra clear here?  I think it's all right either way.

File: themes/default/content/docs/intro/console/organizations.md
Comment (jkisk) [inline themes/default/content/docs/intro/console/organizations.md:107]: I think the original might have been a bit more clear here, I read it to clarify just now.

File: themes/default/content/docs/intro/console/projects-and-stacks.md
Comment (jkisk) [inline themes/default/content/docs/intro/console/projects-and-stacks.md:133]: resources > resource

**Review by jkisk — APPROVED** [review]
These are looking so much better, thanks for taking this on!

**Comment by susanev** [comment]
@cnunciato did you want to re-review this?

**Review by cnunciato — APPROVED** [review]
:shipit:

**Pull request merged**
